data/teamData.js

```diff
@@ -1,69 +1,75 @@
     },
     {
-      name: "Angad Garg",
+      name: "Roy Chiu",
       year: "'20",
       quote:
-        "Hack4Impact has given me the opportunity to apply my skills towards important causes while improving myself at the same time. I hope to help others experience this personal growth, see the impact their work can have, and collaborate with other people who care just as much. ",
-      linkedin: "https://www.linkedin.com/in/angad-garg/"
-    }
+        "I joined Hack4Impact because I wanted to work on impactful products with like-minded, dedicated, and talented people. I hope to give others the same opportunities to work on projects they love with people they love through Hack4Impact.",
+      linkedin: "https://www.linkedin.com/in/royguanchiu/"
+    },
   ],
   team: [
     {
-      name: "Annie Wu",
-      role: "Director of PR",
-      linkedin: "https://www.linkedin.com/in/anniewu701/"
+      name: "Hana Rimawi",
+      role: "Community Director",
+      linkedin: "https://www.linkedin.com/in/hana-rimawi/"
     },
     {
-      name: "Doris Zhao",
-      role: "Community Director"
+      name: "Daniel Choi",
+      role: "Director of ER",
+      linkedin: "https://www.linkedin.com/in/choiboy98/"
     },
     {
-      name: "Hana Rimawi",
+      name: "Annie Wu",
       role: "Product Manager",
-      linkedin: "https://www.linkedin.com/in/hana-rimawi/"
+      linkedin: "https://www.linkedin.com/in/anniewu701/"
     },
     {
-      name: "Shreyas Mohan",
+      name: "Arpan Laha",
       role: "Product Manager",
-      linkedin: "https://www.linkedin.com/in/shreyasmohan/"
+      linkedin: "https://www.linkedin.com/in/arpanlaha/"
     },
     {
-      name: "Arpan Laha",
+      name: "Aryn Harmon",
+      role: "Tech Lead",
+      linkedin: "https://www.linkedin.com/in/aryn/"
+    },
+    {
+      name: "Betty Shen",
       role: "Product Manager",
-      linkedin: "https://www.linkedin.com/in/arpanlaha/"
+      linkedin: "https://www.linkedin.com/in/cybettyshen/"
     },
     {
-      name: "Josh Burke",
+      name: "Alan Fang",
       role: "Product Manager",
-      linkedin: "https://www.linkedin.com/in/josh-burke/"
+      linkedin: "https://www.linkedin.com/in/alan-fang/"
     },
     {
-      name: "Aria Malkani",
+      name: "Josh Byster",
       role: "Tech Lead",
-      linkedin: "https://www.linkedin.com/in/arshia-malkani-4b458010b/"
+      linkedin: "https://www.linkedin.com/in/joshbyster/"
     },
     {
-      name: "Megha Mallya",
+      name: "Kelley Chau",
       role: "Tech Lead",
-      linkedin: "https://www.linkedin.com/in/megha-mallya/"
+      linkedin: "https://www.linkedin.com/in/kelley-chau-838ab313b/"
     },
     {
-      name: "Roy Chiu",
+      name: "Neeraj Aggarwal",
       role: "Tech Lead",
-      linkedin: "https://www.linkedin.com/in/royguanchiu/"
+      linkedin: "https://www.linkedin.com/in/neeraj-aggarwal/"
     },
     {
-      name: "Aryn Harmon",
+      name: "Megha Mallya",
```

Comment:
I don't think Megha is TL right?

Reply:
good point, but I left it just cause she was a TL at some point. Once we move class of 20 to alumni status, it'd probably make more sense to move list her as a TL so I kept it that way here too. Open to disagreement though.